operations/helm/charts/mimir-distributed/values.yaml

```
@@ -846,6 +856,73 @@ query_frontend:
   env: []
   extraEnvFrom: []
 
+query_scheduler:
+  enabled: false
+  replicas: 1
```

I would make this 2 replicas. The [docs](https://grafana.com/docs/mimir/latest/operators-guide/architecture/components/query-scheduler/#operational-considerations) recommend running at least two replicas.   Having only one replica combined with the `maxUnavailable:1, maxSurge:0` means that on upgrades there will be downtime. 

```suggestion
  replicas: 2
```